concordium-std/Cargo.toml

```
@@ -34,6 +39,7 @@ std = ["concordium-contracts-common/std"]
 wasm-test = ["concordium-std-derive/wasm-test"]
 build-schema = ["concordium-std-derive/build-schema"]
 crypto-primitives = ["sha2", "sha3", "secp256k1", "ed25519-zebra"]
+concordium-quickcheck = ["concordium-std-derive/concordium-quickcheck", "getrandom" , "quickcheck"]
```

I think this is the place for enabling `"concordium-contracts-common/concordium-quickcheck"`
```suggestion
concordium-quickcheck = ["concordium-std-derive/concordium-quickcheck", "getrandom", "quickcheck", "concordium-contracts-common/concordium-quickcheck"]
```